junit-runtime/src/main/scala/org/hamcrest/CoreMatchers.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Ported from https://github.com/hamcrest/JavaHamcrest/
+ */
+package org.hamcrest
+
+import org.hamcrest.core._
+
+object CoreMatchers {
+  // Commented matchers where implemented using reflexion. It is possible that
+  // some of them could be reimplemented from scratch without using reflexion.
```

Comment:
Is the word spelled reflexion on purpose?
```suggestion
  // Commented matchers where implemented using reflection. It is possible that
  // some of them could be reimplemented from scratch without using reflection.
```

Reply:
This came straight from Scala.js.

Probably confused with the French version of reflection, which is "réflexion". 😅